### PR TITLE
Create ravost.json

### PR DESCRIPTION
**Pull request by Ravost99**
request for ravost.is-a.dev

### CODE REVIEW

File: domains/ravost.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Ravost99",
+    "discord": "Ravost99#3361"
```
Comment (MaskDuck) [inline domains/ravost.json:4]: email pls?
if dont have / wanna leak, then leave a empty json object like `{"email":""}`

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/ravost.json (hunk 1 above)
Comment (Ravost99) [reply domains/ravost.json:4]: I dont want to share my email is that ok?

File: domains/ravost.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "Ravost99",
+    "email": ""
+    "discord": "Ravost99#3361",
+    "info": "didnt want to share my email"
```
Comment (MaskDuck) [inline domains/ravost.json:6]:
```suggestion
```
this is unneeded

File: domains/ravost.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "Ravost99",
+    "email": ""
```
Comment (MaskDuck) [inline domains/ravost.json:4]:
```suggestion
    "email": "",
```
json thing

**Review by MaskDuck — APPROVED** [review]

**Comment by MaskDuck** [comment]
wait

**Comment by MaskDuck** [comment]
not approve

File: domains/ravost.json (hunk 3 above)
Comment (MaskDuck) [inline domains/ravost.json:4]:
```suggestion
    "email": "",
```
json thing

File: domains/ravost.json (hunk 2 above)
Comment (MaskDuck) [inline domains/ravost.json:6]:
```suggestion
```
not needed

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by MaskDuck — DISMISSED** [review]
not sus

**Review by TheBotlyNoob — DISMISSED** [review]

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**